### PR TITLE
update e2e test

### PR DESCRIPTION
**Pull request by SamYuan1990**
update in e2e test
init for binary build check and key gen function in client
add server client demo for
sm2+sm3 sign verify
sm2 encrypt decrypt
sm4 encrypt decrypt

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

File: workshop/README-zh.md [hunk 1]
```diff
@@ -8,10 +8,22 @@
 
 ## 步骤
 1. 编译项目
+### terminal 1
+`cd server && go build .`
+### terminal 2
+`cd client && go build .`
 1. 生成密钥并配置
```
Comment (xiaohui249) [inline workshop/README-zh.md:15]: 标题序号是否有问题？

File: workshop/README-zh.md [hunk 2]
```diff
@@ -8,10 +8,22 @@
...
+### terminal 2
+`./client ./ generate`
+`mv pub.pem ../server`
 1. 启动服务器
```
Comment (xiaohui249) [inline workshop/README-zh.md:19]: 同上

File: workshop/client/client.go [hunk 3]
```diff
@@ -2,28 +2,115 @@ package main
...
 	"encoding/hex"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
+	"strings"
+	"time"
 
 	"github.com/Hyperledger-TWGC/fabric-gm-plugins/workshop"
```
Comment (xiaohui249) [inline workshop/client/client.go:13]: 在github上没有找到fabric-gm-plugins这个库

**Review by xiaohui249 — CHANGES_REQUESTED** [review]

File: workshop/README-zh.md (hunk 1 above)
Comment (SamYuan1990) [reply workshop/README-zh.md:15]: readme able to use 
1. aaa
1. bbb

equals to

1. aaa
2. bbb

File: workshop/client/client.go (hunk 3 above)
Comment (SamYuan1990) [reply workshop/client/client.go:13]: https://github.com/Hyperledger-TWGC/GM-interoperability/blob/master/interop/go.mod#L1

here your are

**Review by xiaohui249 — APPROVED** [review]

**Pull request merged**